edgecompute/examples/authentication/jwt-signing/README.md

```diff
@@ -0,0 +1,21 @@
+# JWT Signing
+
+_Keyword(s):_ jwt, authentication<br>
+
+## Copyright Notice
+
+    (c) Copyright 2024 Akamai Technologies, Inc. Licensed under Apache 2 license.
+
+This project provides an Akamai Edgeworker solution for signing JSON Web Tokens (JWTs) and attaching them to API requests. It includes an example of how to sign a JWT using HMAC-SHA256 and then add the JWT and an API key to the headers of an outgoing request.
+
+## Limitations
```

Comment:
I should have mentioned this before: this is no longer a limitation, it's now Security Considerations. Please pick that or some other label that reminds users they need to think about the security implications of their key storage.

Reply:
Done.